tests/README.md

```diff
@@ -80,16 +80,15 @@ In addition to standard go tests parameters, the following custom parameters are
  Parameter | Description | Possible values | Default
  --- |--- | --- | ---
 rook_platform| platform rook needs to be installed on  | kubernetes | kubernetes
-k8s_version  | version of Kubernetes to be installed  | v1.5+  | v1.7.5
+k8s_version  | version of Kubernetes to be installed  | v1.6+  | v1.8.5
 rook_image | rook image name to be installed | valid image name | rook/rook
 toolbox_image | toolbox image name to be installed | valid image name | rook/toolbox
 skip_install_rook | skips installing rook (if already installed) | true or false  | false
-load_parallel_runs | performs concurrent operations (optional used for load test) | any number | 20
-
-If the `install_rook` flag is set to false, then all the other flags are ignored
-and tests are run without rook being installed and setup. Use this flag to run tests against
-a pre-installed/configured rook.
-
+load_parallel_runs | performs concurrent operations (load test only) | any number | 20
+load_volumes | number of volumes ( block,object store or shared file systems) to be created during load test (load test only)| >1 | 1
```

Comment:
1. i'd suggest the following spacing:
`(block, object store, or shared file systems)`
2. do you mount object store volumes? or are you referring only to block and file?
3. you can remove "to be created during load test" since you already said that in parenthesis

Reply:
we just mount block and file. I will actually remove file and block, feels redundant.